src/cljs/zetawar/subs.cljs

```diff
@@ -456,8 +455,9 @@
 
 (deftrack compatible-armor-types-for-repair? [conn targeted-q targeted-r]
   (when-let [[selected-q selected-r] @(selected-hex conn)]
-    (game/compatible-armor-types-for-repair? @conn @(game conn) @(unit-at conn selected-q selected-r)
-                                                                @(unit-at conn targeted-q targeted-r))))
+    (game/compatible-armor-types-for-repair? @conn @(game conn)
```

Comment:
There was a problem when doing the checks on circleci that seemed to stem from using :unit-type/armor-type in the subs/unit-at function. Adapting subs_test.cljs to accommodate this wasn't working, so I used game/unit-at instead. This may just be a result of my inexperience with both git and testing frameworks like circleci, so I can try to fix it.